x-pack/test/security_solution_api_integration/common/lib/authentication/roles.ts

```diff
@@ -0,0 +1,511 @@
+/*
```

Comment:
Are these authentication files duplicated from `x-pack/test/rule_registry/common/lib/authentication` ?
Is this possible to have only one copy in common directory?

Reply:
Yes, good catch @vitaliidm , I'll try to reuse them.